libs/core/state/src/operation/state.ts

```diff
@@ -0,0 +1,23 @@
+import { DaffStateError } from '../errors/public_api';
+import { DaffState } from '../states/public_api';
+
+/**
+ * A basic operation state.
+ * Represents the current state of an operation and any associated errors.
+ */
+export interface DaffOperationState {
+  /**
+   * The loading state of the operations backed by this state.
+   */
+  loading: DaffState;
+
+  /**
+   * The errors generated by the most recent operation.
+   */
+  errors: DaffStateError[];
+}
+
+export const daffOperationInitialState: DaffOperationState = {
```

Comment:
Same idea here with code folding, this is purely organizational code.

Reply:
this is used in tests and does have a use case in consuming code. If I were to unexport this, I would just delete the token entirely and inline the code manually.